code/modules/mapping/merge_conflicts.dm

```diff
@@ -0,0 +1,16 @@
+// Used by mapmerge2 to denote the existence of a merge conflict (or when it has to complete a "best intent" merge where it dumps the movable contents of an old key and a new key on the same tile).
+// We define it explicitly here to ensure that it shows up on the highest possible plane (while giving off a verbose icon) to aide mappers in resolving these conflicts.
+// DO NOT USE THIS IN NORMAL MAPPING!!! Linters WILL fail.
+
+/obj/merge_conflict_marker
+	name = "Merge Conflict Marker - DO NOT USE"
+	icon = 'icons/landmarks.dmi'
+	icon_state = "merge_conflict_marker"
+	desc = "If you are seeing this in-game: someone REALLY, REALLY, REALLY fucked up. They physically mapped in a fucking Merge Conflict Marker. What the shit."
+
+///We REALLY do not want un-addressed merge conflicts in maps for an inexhaustible list of reasons. This should help ensure that this will not be missed in case linters fail to catch it for any reason what-so-ever.
+/obj/merge_conflict_marker/Initialize(mapload)
+	. = ..()
+	var/msg = "HEY, LISTEN!!! Merge Conflict Marker detected at [AREACOORD(src)]! Please manually address all potential merge conflicts!!!"
+	to_chat(world, SPAN_BOLDANNOUNCE("[msg]"))
+	warning(msg)
```

Comment:
Although, this doesn't scream loud enough so will go unnoticed until someone actually tests in game and reads chat during startup. This doesn't currently result in a unit test failure; so it would be better if it threw it as a runtime instead of a debug message for check_runtimes to pick up on.

I tried testing two branches off of this branch both editing the basket ball on the Almayer: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/de9bff33-befa-4b20-b036-210b3788228e)
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/53c1c413-7ce2-4d48-a6ad-444c63e928f9)
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/e9cf9bcf-a70b-411b-a7d9-2610fe76fd9b)
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/7cd95836-0a48-405e-9bf3-d4b8369140c0)

```suggestion
	CRASH(msg)
```

If you do above then: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/15f9688f-22d5-4f40-9220-91eee8732e06)

Reply:
> Although, this doesn't scream loud enough so will go unnoticed until someone actually tests in game and reads chat during startup. This doesn't currently result in a unit test failure; so it would be better if it threw it as a runtime instead of a debug message for check_runtimes to pick up on.
> 
> I tried testing two branches off of this branch both editing the basket ball on the Almayer: ![image](https://user-images.githubusercontent.com/76988376/280741226-de9bff33-befa-4b20-b036-210b3788228e.png) ![image](https://user-images.githubusercontent.com/76988376/280743765-53c1c413-7ce2-4d48-a6ad-444c63e928f9.png) ![image](https://user-images.githubusercontent.com/76988376/280743829-e9cf9bcf-a70b-411b-a7d9-2610fe76fd9b.png) ![image](https://user-images.githubusercontent.com/76988376/280743931-7cd95836-0a48-405e-9bf3-d4b8369140c0.png)
> 
> If you do above then: ![image](https://user-images.githubusercontent.com/76988376/280746670-15f9688f-22d5-4f40-9220-91eee8732e06.png)

sure, but it is also picked up in run linters currently